2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 144 Line 2_
```python
# Get a range from 5 to 9
print(list(range(5,10)))
```
If you're sick of all the `print` functions wrapping your code, you can leave them out for the last line of a notebook and still get the output printed in the book

```python
# Get a range from 5 to 9
list(range(5,10))
```

I think  it's good to start with the `print`s though cause the auto-printing trick is only a notebook thing rather than a python one.